universal-login-sdk/test/SdkSigner.ts

```diff
@@ -4,11 +4,13 @@ import {loadFixture} from 'ethereum-waffle';
 import basicWalletService from './fixtures/basicSDK';
 import {SdkSigner} from '../lib/SdkSigner';
 
-describe('SdkSigner', () => {
+describe('E2E: SdkSigner', () => {
```

Comment:
Should we rename `SdkSigner -> SDKSigner` ?

Reply:
Good question, not sure